### PR TITLE
media: Add test coverage for VegaRtcManager

### PR DESCRIPTION
**Pull request by havardholvik**
### Description

We need to start somewhere. This change breaks out the calculation of spatial and temporal layers so they can be unit tested.

### Testing

Automated tests should cover this, no functional change.

### Checklist

-   [x] My code follows the project's coding standards.
-   [x] Prefixed the PR title and commit messages with the service or package name
-   [x] I have written unit tests (if applicable).
-   [ ] I have updated the documentation (if applicable).
-   [x] By submitting this pull request, I confirm that my contribution is made
        under the terms of the MIT license.

### CODE REVIEW

File: packages/media/src/webrtc/VegaRtcManager/utils.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    // todo: consider spatialLayer 1 for maxSide > 200. 4 in a room is considerable higher quality on phones this way
+    // ... but might conflict with our intent of mobile-mode
+    if (numberOfActiveVideos < 4 && maxSide > 300 && spatialLayer === 0) {
+        spatialLayer = 1;
+    }
+
+    return { spatialLayer, temporalLayer };
+}
+
+export function getNumberOfActiveVideos(consumers: any) {
```
Comment (kevinwhereby) [inline packages/media/src/webrtc/VegaRtcManager/utils.ts:41]: What do you think about adding some types as we go?

these two functions are `Consumer[]` and `Consumer` imported from
`import { Consumer } from "mediasoup-client/lib/Consumer";`

Comment (havardholvik) [reply packages/media/src/webrtc/VegaRtcManager/utils.ts:48]: I can do that in a separate PR 👍

Comment (havardholvik) [reply packages/media/src/webrtc/VegaRtcManager/utils.ts:48]: I will do 👍

Comment (havardholvik) [reply packages/media/src/webrtc/VegaRtcManager/utils.ts:48]: Ok, Im not going to add this just now. Adding the types gave a few errors, as we are relying on private variables. I want us to set up an e2e test for stream resolution before I go ahead and change how we access some of this data.

**Pull request merged**